assets/js/modules/thank-with-google/datastore/publications.js

```diff
@@ -16,6 +16,12 @@
  * limitations under the License.
  */
 
+/**
+ * External dependencies
+ */
+import invariant from 'invariant';
+import { addQueryArgs } from '@wordpress/url';
```

Comment:
```suggestion
/**
 * External dependencies
 */
import invariant from 'invariant';

/**
 * WordPress dependencies
 */
import { addQueryArgs } from '@wordpress/url';
```

Reply:
Uh oh! I'm so sorry for overlooking that and thank you for noticing!

Reply:
No worries, all good 👍